app/src/main/res/layout/activity_main.xml

```diff
@@ -5,15 +5,164 @@
     android:id="@+id/main"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@color/white"
     tools:context=".MainActivity">
 
+    <View
+        android:id="@+id/background_my_profile"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@drawable/background"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHeight_percent="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline_start"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_percent="0.05" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline_end"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_percent="0.95" />
+
     <TextView
+        android:id="@+id/text_settings"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
+        style="@style/TextAppearance.AppCompat.White24"
```

Comment:
You are using textSize and fontFamily attributes to configure the appearance, but is not enough for appearance like on design. You might notice text "Settings" on design has the weight attribute in typography section with value of 600. This sets the thickness of the text without changing its size.
You could use the attribute textFontWeight, but it is only available for minSdk 26, so you need to use a [separate font](https://fonts.google.com/specimen/Open+Sans) with a weight already set for this task.

Reply:
done